doc/introduction.rst

```diff
@@ -234,27 +261,29 @@ which will open an interactive prompt::
 
 .. topic:: `>>>` **Prompt**
 
-   Below we'll be using `>>>` to indicate input lines. If you wish to copy
-   these input lines directly into *IPython*, click on the `>>>` located
-   at the top right of the code block
+   Below we'll be using `>>>` to indicate input lines. If you wish to copy and
+   paste these input lines directly into *IPython*, click on the `>>>` located
+   at the top right of the code block to toggle these prompt signs
 
 
 Your first steps with nilearn
 ------------------------------
 
-nilearn does not have a graphical user interface, it is used via Python
-code. Most important function to process neuroimaging data take as an
-input either a filename, or a `NiftiImage object
-<http://nipy.org/nibabel/nibabel_images.html>`_, which we call a
+First things first, nilearn does not have a graphical user interface.
```

Comment:
What about "First things first, nilearn does not have a graphical user interface. But you will soon realize that you don't really need one."?